torch/csrc/jit/mobile/import.cpp

```diff
@@ -375,18 +380,26 @@ void BytecodeDeserializer::parseMethods(
           std::move(*std::move((*debug_handles)[i]).toTuple()).elements();
     }
 
-    parseInstructions(
-        function_name,
-        std::move(ins_list),
-        debug_handles_m_tuple,
-        function.get());
-
+    // 1. First pass all operators from models
     parseOperators(
         std::move(ops_list),
         model_version,
         module_load_options_,
         function.get());
 
+    // 2. Decides if upgrader is needed
+    bool use_upgrader =
```

Comment:
Is it used?

Reply:
It's used in #67731. Would be great to move it there for self-contain.

Reply:
`bool use_upgrader` is a flag and it's decided by the loaded model operator version.. This variable is initialized but not used. One most important change in this pr is to swapping the sequence. Previously `parseInstructions` -> `parseOperators`, now `parseOperators` -> `parseInstructions`.